mmv1/products/biglakeiceberg/product.yaml

```diff
@@ -17,7 +17,7 @@ legacy_name: 'biglake'
 display_name: 'Biglake'
 versions:
   - name: 'ga'
-    base_url: 'https://biglake.googleapis.com/iceberg/v1/'
+    base_url: 'https://biglake.googleapis.com/'
```

Comment:
The URL form for this product is unusual- just confirming explicitly, the iceberg resources are *not* documented on https://docs.cloud.google.com/bigquery/docs/reference/biglake/rest (or seemingly elsewhere on c.g.c). Specifically, the service endpoint (`biglake.googleapis.com`) is typically followed by the API version directly (per https://google.aip.dev/185) where this contains the `iceberg` URL part instead, I assuming denoting the iceberg subcollection

(This isn't very material to the Terraform implementation, and went through earlier review, but was just a bit unusual so wanted to make note!)